src/Components/SuperAdmins/Form/Form.jsx

```diff
@@ -70,72 +83,87 @@ const Form = () => {
       };
       const url = `${process.env.REACT_APP_API_URL}/admins`;
       fetch(url, post).then(async (res) => {
-        const { message } = await res.json();
-        if (res.status !== 200 && res.status !== 201) {
-          alert(message);
+        const { message, error } = await res.json();
+        setModalMessage(message);
+        if (!error) {
+          toggleModal('success');
         } else {
-          alert(message);
-          redirect();
+          toggleModal('error');
         }
       });
     }
   };
 
   return (
-    <section>
-      <h2>Edit Admin</h2>
+    <section className={styles.container}>
+      <Modal
+        showModal={showModal.error}
+        closeModal={() => toggleModal('error')}
+        text={modalMessage}
+        variant="errorModal"
+      />
+      <Modal
+        showModal={showModal.success}
+        closeModal={() => {
+          toggleModal('success');
+          redirect();
+        }}
+        text={modalMessage}
+        variant="successModal"
+      />
+      <h2>{idAdmin ? 'Edit' : 'Create'} Admin</h2>
       <form className={styles.form}>
         <div className={styles.div}>
-          <label>First Name</label>
+          <label className={styles.label}>First Name</label>
           <input
             className={styles.input}
             type="text"
             name="name"
             defaultValue={input.name}
             onChange={onChangeInput}
           />
-          <label>Last Name</label>
+          <label className={styles.label}>Last Name</label>
           <input
             className={styles.input}
             type="text"
             name="lastName"
             defaultValue={input.lastName}
             onChange={onChangeInput}
           />
-          <label>Email</label>
+          <label className={styles.label}>Email</label>
           <input
             className={styles.input}
             type="text"
             name="email"
             defaultValue={input.email}
             onChange={onChangeInput}
           />
-          <label>Password</label>
+          <label className={styles.label}>Password</label>
           <input
             className={styles.input}
             type="password"
             name="password"
             defaultValue={input.password}
             onChange={onChangeInput}
           />
-          <label>Status</label>
+          <label className={styles.label}>Status</label>
           {idAdmin ? (
-            <select name="status" onChange={onChangeInput}>
+            <select name="status" onChange={onChangeInput} className={styles.select}>
               <option value="inactive">Inactive</option>
               <option value="active">Active</option>
             </select>
           ) : (
```

Comment:
```suggestion
          ) :  null}
```

You don't need to show status on create

Reply:
ty Dami! I fix it